### ISSUE TITLE
cusolver error: CUSOLVER_STATUS_EXECUTION_FAILED, when calling `cusolverDnSgetrf( handle, m, n, dA, ldda, static_cast<float*>(dataPtr.get()), ipiv, info)`. This error may appear if the input matrix contains NaN.

### ISSUE THREAD

**Issue by hhuyujie** [yangxy/PASD#7]
I test according to readme.md and report an error.  

"cusolver error: CUSOLVER_STATUS_EXECUTION_FAILED, when calling `cusolverDnSgetrf( handle, m, n, dA, ldda, static_cast<float*>(dataPtr.get()), ipiv, info)`. This error may appear if the input matrix contains NaN."

There is a problem with "latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs, return_dict=False)[0]" in line 1151, pipelines/pipeline_pasd.py.

I hope you can give me some guidance, thank you very much!

**Comment by yangxy** [comment]
Please install the latest torch (2.0.1), torchvision (0.15.2) and cuda (12.1)

**Comment by hhuyujie** [comment]
Thanks for your response. It works, but the results are strange. I run `python test_pasd.py` with your default configuration and get the result shown below.

image_path example/dog.png
![image](https://github.com/yangxy/PASD/assets/47933624/d7d9df52-7b3a-4268-b2f8-937b812e8c06)

output_dir output/dog.png
![dog](https://github.com/yangxy/PASD/assets/47933624/0aa6427d-30e1-4609-b0a2-dc5c505e7b63)

**Comment by yangxy** [comment]
This is weird. Please try to upgrade the latest diffusers and transformers. FYI, I use diffusers==0.20.0 and transformers==4.25.0
